### ISSUE TITLE
Media displaying as links in group chat

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-Android-XMPP#424]
@n8fr8 We're seeing this happen in a newly created group that iOS created. Audio and photos show as links. I'll get a screenshot.

![screenshot_20180205-151112](https://user-images.githubusercontent.com/1434597/35829247-db286d68-0a87-11e8-9e65-418b3c3e0b18.png)

Android Device: Moto X
OS: Android 5.1

Device: Moto G play
OS: Android 6.0.1

Android Device: Samsung Galaxy S7
OS: Android 5.1

App version: 15.4 Alpha 3

**Comment by n8fr8** [comment]
More detail on this?

**Comment by tiffrobo** [comment]
- This is still happening.
- iOS creates the group and adds Android friends. Any media sent from the Android devices to the group results in a link for the other Android devices. 
- iOS media does not produce links. 
- This is isolated to iOS created groups.

Device: iPhone 7
OS: 11.2.2

Device: Moto G play
OS: Android 6.0.1

Device: Moto X 
OS: Android 5.1
Android 15.4 beta 2

**Comment by n8fr8** [comment]
I think the fact it was an iOS group is a red herring. I think is related to something else. I can't reproduce here (Zom on ipod touch created group, added two android friends, everyone sharing media w/o links being displayed)

**Comment by tiffrobo** [comment]
Sometimes occurring! Will try more focused debugging...

**Case: 1** Not working
Steps: iPhone 7 iOS creates the group with two Android devices.
- iOS sends media (photo, audio) to the group. 
- Android receives images and audio just fine.
- Android sends media (photo, audio) to the group. 
- iOS receives images and audio fine.
- Android devices in the group are receiving links from images and audio messages if they aren't the sender. 

**Case 2:** Working
Steps: iPad creates the group with two Android devices.
- iOS sends media (photo, audio) to the group. 
- Android receives images and audio just fine.
- Android sends media (photo, audio) to the group. 
- iOS receives images and audio fine.
- Android devices receive images and audio just fine.

**Case: 3** Working
Steps: iPhone 6 creates the group with two Android devices. 
- iOS sends media (photo, audio) to the group. 
- Android receives images and audio just fine.
- Android sends media (photo, audio) to the group. 
- iOS receives images and audio fine.
- Android devices receive images and audio just fine.

**Android:**
Device: Moto G play
OS: Android 6.0.1

Device: Moto X 
OS: Android 5.1
Build: 15.4.0 RC 1

**iOS:**
Device: iPhone 7
OS: 11.2.2

Device: iPhone 6+
OS: 11.2.5

Device: iPad
OS: 10.3.2
App version: Build 121

**Comment by n8fr8** [comment]
Are you mixing staging and home uses here? There can be confusion if the link is coming from a server other than the users home server. We don't autofetch all links.

**Comment by tiffrobo** [comment]
Ok good point. The iPhone 7 device is on the staging server while **ALL** other devices are on the home server. And we are only seeing the issue in the group created by the device on the staging server.

**Comment by n8fr8** [comment]
Yeah thats it. We can work on this UX in the future, but this is a "feature" of sorts, and not a bug, at this point. Most users, all on home, won't see this.
